src/components/autocomplete/js/autocompleteController.js

```diff
@@ -166,6 +167,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     var wait = parseInt($scope.delay, 10) || 0;
     $attrs.$observe('disabled', function (value) { ctrl.isDisabled = !!value; });
     $attrs.$observe('required', function (value) { ctrl.isRequired = !!value; });
+    $attrs.$observe('readonly', function (value) { ctrl.isReadonly = !!value; });
```

Comment:
Yes, then the attributes will also work with an empty attribute value. And a little side-effect will be, that `required="false"` also works. This won't break any existing versions, because it just adds `false` and `0` as a negative value.